### PR TITLE
Polaris Sync 09-27-2018

### PR DESCRIPTION
**Pull request by Arokha**
*I could be bounded in a nutshell, and count myself king of infinite space, were it not that I had bad line endings.*

------

Well, it compiles locally.
WIP.

### CODE REVIEW

**Comment by SpadesNeil** [comment]
There might be a change somewhere in here that removes phoron off the autofilter for air alarms.

Please revert this change. I still say Polaris is dumb for changing it.

**Comment by KasparoVy** [comment]
Hooooly moly. This one's a doozie

**Comment by Arokha** [comment]
> There might be a change somewhere in here that removes phoron off the autofilter for air alarms.
> Please revert this change. I still say Polaris is dumb for changing it.

I agree, especially given our map and the safety concerns that NT would presumably have here, but not as part of this PR. This is a sync not a "implement feature changes" PR. :V Just make one after, but before update.

**Comment by Anewbe** [comment]
For when you do hunt down that change, it's line 23 in `code/ATMOSPHERICS/components/unary/vent_scrubber.dm`

File: code/ATMOSPHERICS/components/unary/vent_scrubber.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 	var/hibernate = 0 //Do we even process?
 	var/scrubbing = 1 //0 = siphoning, 1 = scrubbing
-	var/list/scrubbing_gas = list("carbon_dioxide", "phoron")
+	var/list/scrubbing_gas = list("carbon_dioxide")
```
[the remark this answers is not in the document]
Comment (SpadesNeil) [reply code/ATMOSPHERICS/components/unary/vent_scrubber.dm]: https://github.com/VOREStation/VOREStation/pull/4296

**Pull request merged**